spec/stackSpec.js

```diff
@@ -4,19 +4,76 @@ import Stack from '../src/stack'
 
 chai.use(chaiChange)
 
-describe('Stack', () => {
+describe.only('Stack', () => {
   'use strict'
 
   it('exists', () => {
     expect(Stack).to.be.a('function')
   })
 
+  context('isEmpty()', () => {
+    it('returns true if the stack is empty', () => {
+      const testEmpty = new Stack()
+
+      expect(testEmpty.isEmpty()).to.equal(true)
+    })
+
+    it('returns false if the stack is not empty', () => {
+      const testEmpty = new Stack()
```

Comment:
I was being consistent with naming my variable with the name of the method and test appended to the front (although that means that this variable should have been 'testIsEmpty').